src/esm_tests/repos.py

```diff
@@ -1,33 +1,56 @@
 import os
 
+from loguru import logger
+
 from .test_utilities import sh
 
+
 def update_resources_submodule(info, verbose=True):
     """
     Initiates and updates the module ``esm_tests_info``.
     """
-    os.chdir(info["script_dir"])
-    sh("git submodule init", verbose=verbose)
-    sh("git submodule update", verbose=verbose)
+    if info["repo_update"]:
+        resources_branch = "release"
+        text = f"Updating the resources repo with '{resources_branch}' branch"
+
+        logger.info(text)
+        logger.info("-" * len(text))
+
+        # Initiate the subodule
+        os.chdir(info["script_dir"])
+        sh("git submodule init", verbose=verbose)
+        sh("git submodule update", verbose=verbose)
+        os.chdir(f"{info['script_dir']}/resources")
+
+        # Pull the desired branch of resources
+        try:
+            sh(f"git checkout {resources_branch}")
+            sh("git pull")
+        except:
+            logger.warning(f"Pull of {resoureces_branch} branch not possible")
```

Comment:
I think this is more severe than a warning, right? Maybe `logger.error`. But then, a follow-up action should be taken. Eg. `sys.exit()`. I don't know about the details much though.
Also, be careful about the typo `resoureces_branch`

Reply:
thanks, I did not know that!